docs/src/developer/standards.rst

```diff
@@ -1,47 +1,86 @@
+by running ``$ tox -e flake8`` and ``$ tox -e pylint``.
 
-To run all of expected linters execute::
+.. _standard-vcs:
 
-   tox -e lint
+Version Control Guidelines
+==========================
 
-.. _flake8: http://flake8.pycqa.org/en/latest/
-.. _pylint: https://www.pylint.org/
-.. _check-manifest: https://pypi.org/project/check-manifest/
-.. _readme_renderer: https://pypi.org/project/readme_renderer/
-.. _PyPI: https://pypi.org/
+To collaborate through VCS, we follow the
+`gitflow <https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow>`_
+workflow. The *develop* and *master* branches are protected and can only be changed with pull
+requests.
+
+For branch names, we recommend prefixing the name of the branch with *feature/*, *fix/*, et
+*doc/* depending on the change you're implementing. Additionally, we encourage adding the issue's id
+at the start of the branch name, after the prefix. For example the branch name for a bug represented
+by issue 225 would be ``fix/225-short-bug-description``.
+
+When creating a release, we use the pattern *release-{version}rc*. This branch represent the release
+candidate that will be merge in the master branch when the changes are ready to be launched in
+production.
+
+Synchronization
+---------------
+Regarding merges, we recommend you keep your changes in your forked repository for as long as
+possible and rebase your branch to Oríon's develop branch before submitting a PR.
+Most probably, the develop branch will have changed by the time your PR is approved. In such cases,
+we recommend to merge the changes from develop to your branch and then merge your branch to develop.
+We discourage rebases after the PR has been submitted as it can cause problems in GitHub's review
+system. On another note, merges are always done with the creation of a merge commit, also known
+as a *non fast-forward merge*.
 
-Fork and Pull Request
-=====================
+In some cases where the PR has small and focused changes contained in one or two commits,
+the contribution may be integrated to the development branch using `squash and merge <https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits>`_ to avoid clutter.
+It is strongly encouraged to make small pull requests.
+They are simpler to implement, easier to integrate and faster to review.
 
-Fork Oríon remotely to your Github_ account now, and start by submitting a
-`Pull Request <https://github.com/epistimio/orion/pulls>`_ to us or by
-discussing an `issue <https://github.com/epistimio/orion/issues>`_ with us.
+.. _standard-documenting:
 
-.. image:: https://img.shields.io/github/forks/epistimio/orion.svg?style=social&label=Fork
-   :target: https://github.com/epistimio/orion/network
+Documenting standard
+====================
+
+Our documentation standard is upheld via doc8_. You can verify your documentation modifications
+by running ``$ tox -e doc8``.
```

Comment:
Maybe add a link to the doc on documenting, so that the contributor reading this section do not feel like this section is incomplete.